tests/MSBuildDeviceIntegration/Tests/PerformanceTest.cs

```diff
@@ -50,11 +50,6 @@ void Profile (ProjectBuilder builder, Action<ProjectBuilder> action, [CallerMemb
 				Assert.Fail ($"No timeout value found for a key of {caller}");
 			}
 
-			if (Builder.UseDotNet) {
-				//TODO: there is currently a slight performance regression in .NET 6
-				expected += 500;
-			}
```

Comment:
Do these still pass? It might mean we would need to bump the times by 500?

Reply:
These are passing with the removal; I think we've overcome this performance loss and it may be worth revisiting these numbers further in the future.

Reply:
I may have spoke too soon, it looks like some of these failed on the latest run. I'll restore this for now.